assets/css/components/sidebar.scss

```diff
@@ -1,11 +1,55 @@
 .sidebar {
+	border-top: 2px solid $blueDark;
+
   .widget {
+	padding-top: $gutterHalf;
     margin-bottom: $gutter;
-    border-top: 4px solid $black;
-    h3 {
-      margin-top: $gutterHalf;
-      margin-bottom: $gutterHalf;
-    }
+	border-top: 1px solid var(--wp--preset--color--mid-grey);
+	
+	&:first-of-type {
+		border-top: none;
+	}
+
+	ol {
+		padding-left: 0;
+	}
+	
+	li,
+	table,
+	p,
+	.wp-block-heading,
+	h3,
+	select {
+		font-size: 1rem;
+	}
+
+	.widget-inner > * > *,
+	.wp-block-latest-comments__comment {
+		margin-bottom: $spacingSmall;
```

Comment:
This margin equates to `5px`, whereas the spacing on GOV.UK is `15px`. The govuk spacing function can be utilised to maintain consistency e.g. `margin-bottom: govuk-spacing(3)`

Reply:
Ok - have adjusted to increase to 15px - it looks a bit loose to me but should be consistent with gov.uk